public/index.css

```diff
@@ -5,6 +5,11 @@
   max-width: 100vw;
   min-width: 100vw;
 }
+input,
+a,
+button {
+  cursor: pointer;
+}
```

Comment:
You can avoid giving styling directly to element

Reply:
its has to be a part of reset CSS. Will Fix it.